tests/script/test_add_preload_script.py

```diff
@@ -95,77 +112,42 @@ async def test_preloadScript_add_multipleScripts(websocket, context_id, html):
         websocket, {
             "method": "script.addPreloadScript",
             "params": {
-                "functionDeclaration": "() => { window.foo1=1; }",
+                "functionDeclaration": "() => { "
+                                       "window.SOME_VAR=["
+                                       "...(window.SOME_VAR ?? []), "
+                                       "'PRELOAD_SCRIPT_1']; }",
             }
         })
     await execute_command(
         websocket, {
             "method": "script.addPreloadScript",
             "params": {
-                "functionDeclaration": "() => { window.foo2=2; }",
-            }
-        })
-
-    await goto_url(websocket, context_id, html())
-
-    result = await execute_command(
-        websocket, {
-            "method": "script.evaluate",
-            "params": {
-                "expression": "window.foo1 + window.foo2",
-                "target": {
-                    "context": context_id
-                },
-                "awaitPromise": True,
-                "resultOwnership": "root"
-            }
-        })
-    assert result["result"] == {"type": "number", "value": 3}
-
-
-@pytest.mark.asyncio
-async def test_preloadScript_add_sameScript_multipleTimes(
-        websocket, context_id, html):
-    EXPRESSION = "() => { window.foo1 = (window.foo1 ?? 0) + 1; }"
-
-    result = await execute_command(
-        websocket, {
-            "method": "script.addPreloadScript",
-            "params": {
-                "functionDeclaration": EXPRESSION,
-            }
-        })
-    assert result == {'script': ANY_UUID}
-    id1 = result['script']
-
-    result = await execute_command(
-        websocket, {
-            "method": "script.addPreloadScript",
-            "params": {
-                "functionDeclaration": EXPRESSION,
+                "functionDeclaration": "() => { "
```

Comment:
Please re-add this assertion:

```
assert id1 != id2
```

The preload scripts should have different IDs.

Reply:
done